app/src/main/java/com/example/flixster/LogInActivity.java

```diff
@@ -0,0 +1,79 @@
+package com.example.flixster;
+
+import androidx.annotation.NonNull;
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.example.flixster.databinding.ActivityLogInBinding;
+import com.google.android.gms.tasks.OnCompleteListener;
+import com.google.android.gms.tasks.Task;
+import com.google.firebase.auth.AuthResult;
+import com.google.firebase.auth.FirebaseAuth;
+
+public class LogInActivity extends AppCompatActivity {
+
+    private EditText etEmail;
+    private EditText etPassword;
+    private Button btnLogin;
+    private TextView tvRegister;
+    private String email;
+    private String password;
+    private FirebaseAuth fAuth;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        ActivityLogInBinding binding = ActivityLogInBinding.inflate(getLayoutInflater());
+        setContentView(binding.getRoot());
+        getSupportActionBar().hide();
```

Comment:
I think you can change visibility in the xml file, but I'm not sure if this code belongs in the activity or in the xml file